### PR TITLE
Add linting & static checks

### PR DESCRIPTION
**Pull request by KellerFuchs**
- [x] Add Tox configuration calling into `flake8`, `pylint`, and `pydocstrings`.
- [x] Fix syntax warts found by `flake8`.
- [x] Fix issues uncovered by `pylint`.
- [x] Add docstrings.
- [x] Add mypy-powered static type checks.

### CODE REVIEW

**Comment by KellerFuchs** [comment]
@kaniini I'm doing that as a prelude to trying to use `python-ssh2` (as I find it much easier to make that sort of changes if I have tooling that picks up simple-ish mistakes)

File: antissh.py [hunk 1]
```diff
@@ -27,7 +30,11 @@
 NICKNAME = config.get('host', 'nickname', fallback='antissh')
 SERVER_PASSWORD = config.get('host', 'password', fallback=None)
 MODES = config.get('host', 'modes', fallback='')
-KLINE_CMD_TEMPLATE = config.get('host', 'kline_cmd', fallback='KLINE 86400 *@{ip} :Vulnerable SSH daemon found on this host.  Please fix your SSH daemon and try again later.\r\n')
+KLINE_CMD_TEMPLATE = config.get(
+    'host', 'kline_cmd',
+    fallback='KLINE 86400 *@{ip} :Vulnerable SSH daemon found on this host.  '
+             'Please fix your SSH daemon and try again later.\r\n',
```
Comment (equinox) [inline antissh.py]: Is it intended for this to be in two parts now?

**Comment by kaniini** [comment]
I don't understand why we would use python-ssh2?  this entire application is built on top of asyncio and that would be even worse performance?

**Comment by KellerFuchs** [comment]
@kaniini Oh, I misunderstood the lack of answer on IRC as signifying you had no opposition to trying it. python-ssh2 is still usable asynchronously, and it's 5-10× faster than asyncssh (mostly because it's a wrapper around libssh2, which is written in C).
I suggested that as a way to address the performance issue mentionned in the README.

If you think it's a bad idea, I just won't do it  :3

**Comment by KellerFuchs** [comment]
PS: Regardless of whether python-ssh2 turns out to be a better ssh library for antissh, good linting and tests might be useful (and uncovered some warts), so I'd really want your opinion/feedback regardless.

File: antissh.py (hunk 1 above)
Comment (KellerFuchs) [reply antissh.py]: 2 string literals one after the other just get concatenated, so it's the same string, just not as a hugely-long line.

**Comment by kaniini** [comment]
I have no objection to this PR then.

**Comment by KellerFuchs** [comment]
@kaniini I think this should be good to go, if you are OK with it.
You (or your favorite CI service) can run `tox` to run the checks; there are still a couple of things flagged by `pylint`, but I think they are false positives (with the exception of TODO).

**Pull request merged**